### PR TITLE
Ephemeral environments: add build and PR check scripts

### PR DESCRIPTION
**Pull request by gkarat**
Implements https://issues.redhat.com/browse/OCPADVISOR-25 and follows https://consoledot.pages.redhat.com/docs/dev/getting-started/frontend-migration.html#_step_3_add_build_and_pr_check_scripts.

### CODE REVIEW

File: build_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="ocp-advisor"
+# IMAGE should match the quay repo set by app.yaml in app-interface
+export IMAGE="quay.io/cloudservices/ocp-advisor-frontend"
+export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build's workspace
+export APP_ROOT=$(pwd)
```
Comment (ikerreyes) [inline build_deploy.sh:11]: I now it is what is on the docs but sound a bit weird to define a variable after using it as fallback for `WORKSPACE`. Can we double check with other apps?

File: pr_check.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build's workspace
+export APP_ROOT=$(pwd)
+# 16 is the default Node version. Change this to override it.
+export NODE_BUILD_VERSION=16
+COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+IQE_PLUGINS="ocp_advisor"
```
Comment (ikerreyes) [inline pr_check.sh:19]: This will be `ccx`.

File: pr_check.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+IQE_MARKER_EXPRESSION="smoke"
```
Comment (ikerreyes) [inline pr_check.sh:20]: This will be `ui` or `ui and smoke` or something. We will need to decide. But it can be done later on.

**Review by ikerreyes — APPROVED** [review]

**Review by mkholjuraev — APPROVED** [review]
LGTM. All of these are correct according to the docs.

File: build_deploy.sh (hunk 1 above)
Comment (gkarat) [reply build_deploy.sh:11]: this is indeed strange. in other apps we have the same order: https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/build_deploy.sh, https://github.com/RedHatInsights/vulnerability-ui/blob/master/build_deploy.sh for instance

Comment (ikerreyes) [reply build_deploy.sh:11]: Yes, well, maybe it is not used if `WORKSPACE` is always defined...

File: pr_check.sh (hunk 3 above)
Comment (gkarat) [reply pr_check.sh]: ok, I left just `ui`, is it ok for now?

Comment (ikerreyes) [reply pr_check.sh]: sure, nothing will be run, so it really doesn't matter (yet)

**Pull request merged**